extensions-contrib/azure-extensions/pom.xml

```diff
@@ -66,6 +66,49 @@
             <artifactId>jackson-module-guice</artifactId>
             <version>${jackson.version}</version>
         </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <scope>provided</scope>
```

Comment:
Is there a reason to explicitly specify the `provided` deps? (I'm thinking they won't be packaged anyway)

Reply:
The `dependency:analyze` goal (via the `outputXML` property` suggested the `provided` scope.